modules/experimental/push-imagelist.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+. ./defaults.sh
+. ../../include/common.sh
+. .envrc
+
+info "Pushing all imagelist.txt images to local-registry.default.svc.cluster.local"
+
+kubectl create namespace push-imagelist 2>/dev/null || true
+
+# POSIX compliant:
+while IFS= read -r imagelist_file
+do
+    # Create a kube job per image in imagelist.txt. There, pull it, retag it,
+    # and push the image to the local registry
+    for SOURCE_IMAGE in $(cat "$imagelist_file"); do
+        JOB_NAME=${SOURCE_IMAGE%%\:*} # remove suffix after ':'
+        JOB_NAME=${JOB_NAME////-} # substitute '/' with '-'
+        JOB_NAME=${JOB_NAME:0:63} # truncate to 63 charts for kubernetes
+        TARGET_REGISTRY='local-registry.default.svc.cluster.local'
+        kubectl apply --overwrite=false -f - <<HEREDOC
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: ${JOB_NAME}
+  namespace: push-imagelist
+spec:
+  template:
+    spec:
+      containers:
+      - name: copy-image
+        image: dragonchaser/opensuse-skopeo:latest
```

Comment:
This image doesn't have the labels to indicate where the source comes from; we should probably add that.  I assume that's `modules/experimental/docker/Dockerfile.skopeo` though?